setup.py

```diff
@@ -10,12 +10,12 @@
     license='MIT',
     install_requires=['websockets==10.3', 'reactivex==4.0.*'],
     test_suite='tests',
-    url='https://github.com/binary-com/python-deriv-api',
+    url='https://github.com/deriv-com/python-deriv-api',
     project_urls={
-        'Bug Tracker': "https://github.com/binary-com/python-deriv-api/issues",
-        'Documentation': "https://binary-com.github.io/python-deriv-api",
-        'Source Code': "https://github.com/binary-com/python-deriv-api",
-        'Changelog': "https://github.com/binary-com/python-deriv-api/blob/master/CHANGELOG.md"
+        'Bug Tracker': "https://github.com/deriv-com/python-deriv-api/issues",
+        'Documentation': "https://deriv-com.github.io/python-deriv-api",
```

Comment:
@mariovw-deriv is it right? we have used two different endpoint. https://deriv-com.github.io (not accessible) or https://github.com/deriv-com/

Reply:
But this https://deriv-com.github.io/python-deriv-api/ is accessible. Is it a redirect?

Reply:
Yes, trying to redirect to deriv.com. 
<img width="1728" alt="Screenshot 2024-10-30 at 4 33 42 PM" src="https://github.com/user-attachments/assets/4b076b8d-b6f5-406f-a7f7-53983b0a4b75">